app/assets/javascripts/select-kit/addon/components/composer-actions.js

```diff
@@ -130,7 +130,9 @@ export default DropdownSelectBoxComponent.extend({
     if (
       this.action !== CREATE_TOPIC &&
       this.action !== CREATE_SHARED_DRAFT &&
-      !(this.action === REPLY && this.topic && this.topic.isPrivateMessage) &&
+      this.action === REPLY &&
+      this.topic &&
+      !this.topic.isPrivateMessage &&
```

Comment:
Do we have a test that covers this, i.e. that shows the `Reply as Linked Topic ` for non-PMs?

Reply:
We have

https://github.com/discourse/discourse/blob/5238f6788cd42f3ad13c34cb1a04c5073184766b/app/assets/javascripts/discourse/tests/acceptance/composer-actions-test.js#L231-L245

which checks that only the create topic option is present!

Reply:
Ok, sounds good!